sync_diff_inspector/diff.go

```diff
@@ -211,7 +212,6 @@ func (df *Diff) AdjustTableConfig(cfg *Config) (err error) {
 					Schema: schemaTables.Schema,
 					Table:  tableName,
 				},
-				IgnoreColumns:   make([]string, 0, 1),
```

Comment:
do not init it anymore? init directly in L261?

Reply:
yes, in fact golang will initial it